src/controllers/user.controller.ts

```diff
@@ -35,6 +50,22 @@ export class UserController extends BaseController {
             ...req.body
         }
         const result = await this._userService.register(userRegisterDTO);
+        res.status(201).send(result);
+    }
+
+    public async login(req: Request<{}, {}, any>, res: Response): Promise<void> {
+        const userLoginDTO: UserLoginDTO = {
+            ...req.body
+        }
+        const result = await this._userService.login(userLoginDTO);
```

Comment:
there is no checking on result, why do you send 200 status code, perhaps something is wrong

Reply:
this._userService.login throws errors. they are catched in decorator and handled in express error handler.
![image](https://user-images.githubusercontent.com/87005572/203524627-38758e82-7201-4a98-91c2-7eeb684efd00.png)
![image](https://user-images.githubusercontent.com/87005572/203525559-09a04076-9a5c-4433-953e-b0bd64f7ada4.png)
![image](https://user-images.githubusercontent.com/87005572/203525816-3411f16a-2af3-4f01-8d0a-30262de2afb5.png)